src/components/Accordion/Accordion.js

```diff
@@ -3,39 +3,25 @@
  * @copyright IBM Security 2019
  */
 
-import classnames from 'classnames';
 import PropTypes from 'prop-types';
 import React from 'react';
 
 import CarbonAccordion from 'carbon-components-react/lib/components/Accordion';
 
-import { getComponentNamespace } from '../../globals/namespace';
-
-const namespace = getComponentNamespace('accordion');
-
 const { defaultProps, propTypes } = CarbonAccordion;
 
 const Accordion = ({ className, align, ...other }) => (
-  <CarbonAccordion
-    className={classnames(namespace, className, {
-      [`${namespace}--${align}`]: align,
-    })}
-    {...other}
-  />
+  <CarbonAccordion {...other} align={align} />
```

Comment:
👏 Can we remove this component entirely and just refer to Carbon's?

Reply:
No we still need `align="start"` by default, the opposite of Carbon's default.

Reply:
Oh! Good point!